telemetry/specs/telemetry_report.mdk

```diff
@@ -473,48 +548,38 @@ particular deployment.
         [github.com/p4lang/switch](https://github.com/p4lang/switch/blob/master/p4src/includes/drop_reason_codes.h).
 
     See the INT specification [^INT] for definitions of the remaining metadata.
+    
+* Domain Specific ID (16b)
+  : The unique ID of the INT Domain
+
+* DSMdbit (16b): Domain Specific Md Bits
+  : Bitmap that indicates which optional domain specific metadata is 
+    present in the record of the telemetry report. Each bit represents 
+    4 octets of domain specific optional metadata.
```

Comment:
Did you want to nail this down to 4 octets each?
Or leave it flexible and rely on MD Length to pad if necessary?

Reply:
I am ok with 4-octets each. Do we need a wider discussion?

Reply:
We agreed to make it multiple of 4-octet each in the email thread discussion.